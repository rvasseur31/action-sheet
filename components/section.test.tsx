import React from 'react';
import renderer from 'react-test-renderer';
import Section from './section';

describe('render logItem components', () => {
  it('should render properly wihout any props', () => {
    const tree = renderer.create(<Section title="Action sheet" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
