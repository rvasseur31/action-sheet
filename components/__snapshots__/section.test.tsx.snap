// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render logItem components should render properly wihout any props 1`] = `
<View
  style={
    Object {
      "marginTop": 32,
      "paddingHorizontal": 24,
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "fontSize": 24,
          "fontWeight": "600",
        },
        Object {
          "color": "#000",
        },
      ]
    }
  >
    Action sheet
  </Text>
  <Text
    style={
      Array [
        Object {
          "fontSize": 18,
          "fontWeight": "400",
          "marginTop": 8,
        },
        Object {
          "color": "#444",
        },
      ]
    }
  />
  <View
    accessibilityRole="button"
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "margin": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Menu
      </Text>
    </View>
  </View>
</View>
`;
